src/Illuminate/Database/Eloquent/Relations/Relation.php

```diff
@@ -302,6 +302,16 @@ public function relatedUpdatedAt()
         return $this->related->getUpdatedAtColumn();
     }
 
+    /**
+     * Normalize the dictionary key.
+     * @param  mixed  $key - int | string
+     * @return mixed - int | string
+     */
```

Comment:
This PHPDoc block should probably look like this
```php
/**
 * Normalize the dictionary key.
 *
 * @param  string|mixed
 * @return string|mixed
*/
```
Those comments are redundant.